packages/components/API.md

```
@@ -0,0 +1,183 @@
+## Functions
+### **CloseConnection** - Renders a WebSocket close connection method with optional pre and post execution logic.
+### Parameters
+| Name | Type | Description |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)